pkg/cmd/release/list/options.go

```diff
@@ -0,0 +1,42 @@
+package list
+
+import (
+	"fmt"
+
+	"gopkg.in/yaml.v3"
+
+	"kusionstack.io/kusion/pkg/backend"
+)
+
+type Options struct {
+	Backend string
+}
+
+func NewOptions() *Options {
+	return &Options{}
+}
+
+func (o *Options) Validate(args []string) error {
+	if len(args) != 0 {
+		return ErrNotEmptyArgs
+	}
+	return nil
+}
+
+func (o *Options) Run() error {
+	storage, err := backend.NewWorkspaceStorage(o.Backend)
```

Comment:
This command should list all available `Releases`, not the workspace. You can refer to `pkg/cmd/release/unlock.go:111` to get the `ReleaseStorage`